src/debug/debugAdapterNamedPipeServerDescriptorFactory.ts

```diff
@@ -17,60 +18,67 @@ export class DebugAdapterNamedPipeServerDescriptorFactory implements vscode.Debu
     async createDebugAdapterDescriptor(session: vscode.DebugSession, _executable: vscode.DebugAdapterExecutable | undefined): Promise<vscode.ProviderResult<vscode.DebugAdapterDescriptor>> {
         const pipeName =
             session.configuration.pipeName ?? getDebuggerPipeName();
+        const origin =
```

Comment:
Make sure to add `fromCommand` back to the vscode.debug.startDebugging invocations.

Reply:
How would we know if this should be true or false if its not passed into `cleanconfigurewithdebugger` etc? Or is it only true from `doConfigure` in cmakeFileApiDriver?

Reply:
That's the part that we need to change, we should make sure that in any of our commands we're setting `fromCommand` to true. That change won't be in this code, it will be where the commands are invoked, in `extension.ts`, all of the places where we call `vscode.debug.startDebugging`. 

Additionally, we should be removing the `vscode.debug.startDebugging` from the `doConfigure` in cmakeFileApiDriver, because we no longer are using that strategy.